filebeat/input/file/state.go

```diff
@@ -8,41 +8,41 @@ import (
 	"github.com/elastic/beats/libbeat/logp"
 )
 
-// FileState is used to communicate the reading state of a file
-type FileState struct {
+// State is used to communicate the reading state of a file
+type State struct {
```

Comment:
at some point I'd like to turn State into some very minimal interface. different prospector types might have different needs.